### DISCUSSION TITLE
Should saddle switch to luhenry/netlib

### DISCUSSION THREAD

**Discussion by tnielens** [pityka/saddle#395] [General]
Spark and Breeze have switched from fommil/netlib-java to luhenry/netlib. Should saddle follow suit?

See https://issues.apache.org/jira/browse/SPARK-35295 .

**Comment by pityka** [comment]
Yes I think we should upgrade. I especially like the Vector api fallback on jdk16+ there. Also it is maintained. 


Note that saddle's exposure in saddle-linalg is considerable.
